### PR TITLE
Made default views.

### PR DESCRIPTION
**Pull request by QuartzBrandi**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,9 +2,12 @@
   root 'home#index'
 
   resources :users, only: [:show] do
-    GET 'dash', on: :member
+    get 'dash', on: :member
   end
 
+  resources :sessions, only: [:create, :destroy]
```
Comment (catchingash) [inline config/routes.rb:8]: I think we might want to code the specific paths, so that we can use `login_path`?

File: config/routes.rb [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
+  get 'register', to: "sessions#new"
+
   resources :cookbooks # cookbooks will always be attached to a user
     # index, show, new, create, edit, update, destroy
   resources :recipes do # recipes aren't necessarily attached to a cookbook
```
Comment (catchingash) [inline config/routes.rb:13]: I like to add blank lines between the different resources.

File: config/routes.rb (hunk 1 above)
Comment (catchingash) [inline config/routes.rb:8]: Just kidding. I like your idea to have the login form on all pages.

File: config/routes.rb (hunk 2 above)
Comment (QuartzBrandi) [inline config/routes.rb:13]: GO FOR IT.

**Pull request merged**